setup.sh

```diff
@@ -2,3 +2,4 @@
 
 source env/bin/activate
 python -m pip install -r requirements.txt
+pip install git+https://github.com/glyh/qrterm
```

Comment:
this should probably be `python -m pip install` since by default on rpi you only have pip access via python -m pip